riscv-runtime/rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
 channel = "nightly-2024-08-01"
-targets = ["riscv32imac-unknown-none-elf"]
+targets = ["riscv32imac-unknown-none-elf", "riscv32im-risc0-zkvm-elf"]
```

Comment:
is this used only by tooling to ensure both are installed? or does the crate get compiled to both in a `cargo build`?

Reply:
The targets does not affect `cargo build`, only the channel (toolchain) does. In fact, I don't know how the targets are used here. I thought it would be in `rust-analyzer`, but apparently not.